site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,106 @@
 title: 'Automatic annotation'
 linkTitle: 'Automatic annotation'
 weight: 16
-description: 'Guide to using the automatic annotation of tasks.'
+description: 'Automatic annotation of tasks'
 ---
 
-Automatic Annotation is used for creating preliminary annotations.
-To use Automatic Annotation you need a DL model that can be deployed by a CVAT administrator.
-You can find the list of available models in the `Models` section.
+Automatic Annotation in CVAT is a tool that you can use
+to automatically pre-annotate your data with pre-trained models.
 
-1. To launch automatic annotation, you should open the dashboard and find a task which you want to annotate.
-   Then click the `Actions` button and choose option `Automatic Annotation` from the dropdown menu.
+You can also **add models** from popular platforms
+like **Hugging Face** or **Roboflow** to expand the
+range of available options.
+
+> Note: Automatic Annotation is a [paid CVAT Cloud feature](https://www.cvat.ai/pricing/cloud).
+
+See:
+
+- [Running Automatic annotation](#running-automatic-annotation)
+- [Labels matching](#labels-matching)
+- [Models](#models)
+- [Adding models from Hugging Face and Roboflow](#adding-models-from-hugging-face-and-roboflow)
+
+## Running Automatic annotation
+
+To start automatic annotation, do the following:
+
+1. On the top menu, click **Tasks**.
+2. Find the task you want to annotate and click **Action** > **Automatic Annotation**.
 
    ![](/images/image119_detrac.jpg)
 
-1. In the dialog window select a model you need. DL models are created for specific labels, e.g.
-   the Crossroad model was taught using footage from cameras located above the highway and it is best to
-   use this model for the tasks with similar camera angles.
-   If it's necessary select the `Clean old annotations` checkbox.
-   Adjust the labels so that the task labels will correspond to the labels of the DL model.
-   For example, let’s consider a task where you have to annotate labels “car” and “person”.
-   You should connect the “person” label from the model to the “person” label in the task.
-   As for the “car” label, you should choose the most fitting label available in the model - the “vehicle” label.
-   If the chosen model supports automatic attributes detection
-   (like facial expressions, for example: ``serverless/openvino/omz/intel/face-detection-0205``),
-   you can also map attributes between the DL model and your task.
-   The task requires to annotate cars only and choosing the “vehicle” label implies annotation of all vehicles,
-   in this case using auto annotation will help you complete the task faster.
-   Click `Submit` to begin the automatic annotation process.
+3. In the Automatic annotation dialog, from the drop-down list, select [model](#models).
```

Comment:
```suggestion
3. In the Automatic annotation dialog, select a [model](#models) from the drop-down list.
```

Reply:
Please, do not make such a changes to the text. By the tech doc standards, first you tel where you make an action and only then you name the action itself

Reply:
Okay, but it should still say "a model".